src/test/java/BaseTest.java

```diff
@@ -10,4 +30,93 @@ public static void chromeConfigs() {
         }
     }
 
+    public static void clickSubmitBtn() {
+        WebElement loginButton= BaseTest.driver.findElement(By.cssSelector("button[type='submit']"));
+        loginButton.click();
+    }
+
+    public static void providePassword(String password) {
+        WebElement passwordField = BaseTest.driver.findElement(By.cssSelector("[type = 'password']"));
+        passwordField.click();
+        passwordField.sendKeys(password);
+    }
+
+    public static void provideEmail(String email) {
+        WebElement emailField = BaseTest.driver.findElement(By.cssSelector("[type = 'email']"));
+        emailField.click();
+        emailField.sendKeys(email);
+    }
+
+    @AfterMethod
+    public static void tearDownBrowser() {
+        driver.quit();
+    }
+
+    protected static void validation() throws InterruptedException {
+
+        WebElement myPlaylist = driver.findElement(By.cssSelector("a[href*='playlist/24815']"));
+        myPlaylist.click();
+        Thread.sleep(2000);
+        WebElement songWasAdded = driver.findElement(By.xpath("//tr[@class='song-item']//td[contains(text(),'Epic Song')]"));
+        Assert.assertTrue(songWasAdded.isDisplayed());
+    }
+
+    protected static void abdisPlaylist() throws InterruptedException {
+
+        WebElement myPlaylist = driver.findElement(By.xpath("//section[@id='songsWrapper']//li[5]"));
+        myPlaylist.click();
+        Thread.sleep(2000);
+    }
+
+    protected static void addSongToPlaylistBtn() throws InterruptedException {
+
+        WebElement addingTheSong = driver.findElement(By.xpath("//button[@class='btn-add-to']"));
+        addingTheSong.click();
+        Thread.sleep(2000);
+    }
+
+    protected static void addingNewSong() throws InterruptedException {
+
+        WebElement choiceOfSong = driver.findElement(By.xpath("//tr[@class='song-item']//td[contains(text(),'Epic Song')]"));
+        choiceOfSong.click();
+        Thread.sleep(2000);
+    }
+
+    protected static void allSongsTab() throws InterruptedException {
+        WebElement songsTab = driver.findElement(By.xpath("//a[@class='songs']"));
+        songsTab.click();
+        Thread.sleep(2000);
+    }
+
+    protected static void songBeginsToPlay() throws InterruptedException {
+        WebElement songPlays = driver.findElement(By.xpath("//*[@id=\"progressPane\"]/h3"));
```

Comment:
According to such the locator `//*[@id=\"progressPane\"]/h3`  , the song may not play

![image](https://user-images.githubusercontent.com/104761482/197945606-4664d11a-2ea8-4005-9427-9ffa9e1cfaac.png)